dask/dataframe/shuffle.py

```diff
@@ -354,13 +354,18 @@ def _cull_dependencies(self, keys, parts_out=None):
         Does not require graph materialization.
         """
         deps = defaultdict(set)
+        shuffle_group_name = "group-" + self.name
```

Comment:
Since this variable is only used in one place, would you mind inlining it there? It's nice to minimize the number of variables we have to keep track of in our heads : )